packages/manager/apps/dedicated/client/app/ip/components/list/organisation/organisation.routing.js

```diff
@@ -1,16 +1,22 @@
 import controller from './ip-ip-organisation.controller';
 import template from './ip-ip-organisation.html';
+import { listRouting } from '../../../ip.routing';
 
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state('app.ip.dashboard.organisation', {
+  $stateProvider.state('app.ip.organisation', {
     url: '/organisation',
-    template,
-    controller,
-    controllerAs: '$ctrl',
+    views: {
+      ipview: {
+        template,
+        controller,
+        controllerAs: '$ctrl',
+      },
```

Comment:
i am not able to see where you use the `ipview` ?
if it's rendred in ui-view so prefer to let a default behavior or indicate in html where we have to use it.
it's up to you.

Reply:
HI Mohammed, Its been used here: packages/manager/apps/dedicated/client/app/ip/ip.html

Reply:
Ok got it, thanks for replay